src/server/session.ts

```diff
@@ -606,7 +606,7 @@ namespace ts.server {
         protected projectService: ProjectService;
         private changeSeq = 0;
 
-        private updateGraphDurationMs: number | undefined;
+        private performanceData: protocol.PerformanceData | undefined;

```

Comment:
@amcasey, you might be interested in the minor changes to perf reporting I made in this file after our conversation

Reply:
LGTM.  You'll want to create bugs against the editors to start reporting corresponding telemetry.